examples/Cliner/download_model.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+wget -P ../CliNER/models \ https://drive.google
```

Comment:
Will these files be deleted in the future otherwise we can move it to a permanent place.

In addition, I think it is better to upload all binary files here, any pickle or wheel files.